rmgpy/molecule/kekulize.pyx

```diff
@@ -52,12 +52,13 @@ and the process continues until the entire molecule can be solved.
 
 import logging
 
+from .graph cimport Graph, Vertex, Edge
```

Comment:
Can you change this to `from rmgpy.molecule.graph cimport Graph, Vertex, Edge` ?

Reply:
Done!